app/cmd/rpc/mesh/fullnode.go

```diff
@@ -249,7 +267,10 @@ func createNode(urlStr, name string) *fullNode {
 		URL:       urlStr,
 		Servicers: xsync.NewMapOf[*servicer](),
 		Status:    nil,
-		Crons:     nodeCronJobsWorker,
+		// just a default because on each check request the node will return the right value for the height returned
+		// TODO: implement a default from config as "fallback" before get the real node value
+		BlocksPerSession: 4,
```

Comment:
that is added to the check endpoint so we do not need another call to node, reuse the same, that one is a "safety" default value in case it is required before we get the first status check from node. It should not be required because the connectivity check run sync before mesh start servicing.